README.md

```diff
@@ -24,8 +24,8 @@ Empty | Conversation
 ## Requirements
 
 - iOS 12.0+
-- Xcode 14+
-- Swift 5+
+- Xcode 15+
```

Comment:
Because of the swift 3.9 requirement I do need to also upgrade the min XCode version;